### PR TITLE
Add empty amp-form service to kick off work on validation part

### PR DESCRIPTION
**Pull request by mkhatib**
ITI: #3343 

Validation: #3348

### CODE REVIEW

File: extensions/amp-form/0.1/amp-form.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * limitations under the License.
+ */
+
+import {isExperimentOn} from '../../../src/experiments';
+import {getService} from '../../../src/service';
+
+
+function installAmpForm(win) {
+  if (isExperimentOn(win, TAG)) {
+    return getService(win, 'amp-form', () => {
```
Comment (dvoytenko) [inline extensions/amp-form/0.1/amp-form.js]: We need to install service in either case to ensure that nothing in runtime waits for it.

**Comment by dvoytenko** [comment]
LGTM, with one comment

File: extensions/amp-form/0.1/amp-form.js (hunk 1 above)
Comment (mkhatib) [inline extensions/amp-form/0.1/amp-form.js]: Done.

**Comment by mkhatib** [comment]
Thanks Done!

**Pull request merged**